projects/docs/src/app/features/components/dialog-page/dialog-page.component.ts

```diff
@@ -0,0 +1,371 @@
+/* eslint-disable @angular-eslint/component-max-inline-declarations */
+
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Inject,
+  TemplateRef,
+} from '@angular/core'
+import { FormControl, ReactiveFormsModule } from '@angular/forms'
+import { CkButton } from '@corekit/ui/button'
+import {
+  CK_DIALOG_DATA,
+  CkDialog,
+  CkDialogActions,
+  CkDialogClose,
+  CkDialogContent,
+  CkDialogHeader,
+  CkDialogRef,
+  CkDialogSubtitle,
+  CkDialogTitle,
+} from '@corekit/ui/dialog'
+import { CkFormField } from '@corekit/ui/form-field'
+import { CkInput } from '@corekit/ui/input'
+import { CkLabel } from '@corekit/ui/label'
+import { CkP } from '@corekit/ui/typography'
+import { firstValueFrom } from 'rxjs'
+
+@Component({
+  selector: 'simple-dialog',
+  standalone: true,
+  imports: [
+    CkDialogHeader,
+    CkDialogTitle,
+    CkDialogSubtitle,
+    CkDialogContent,
+    CkDialogActions,
+    CkButton,
+    CkDialogClose,
+  ],
+  template: `
+    <ck-dialog-header>
+      <ck-dialog-title>Notifications</ck-dialog-title>
+      <ck-dialog-subtitle>You have 3 unread messages.</ck-dialog-subtitle>
+
+      <button
+        type="button"
+        ckDialogClose
+        displayAsIcon
```

Comment:
@dhutaryan, I'm not very sure about naming of this input. Any suggestions how we can make it better?

Reply:
Is it actual question?